package.json

```diff
@@ -143,7 +143,7 @@
     "reflect-metadata": "^0.1.13",
```

Comment:
It won't let me add a comment up there ^^, but for this release we should bump "version" from "2.1.4" to "2.2.0" 

Note that the data-version is already updated in develop: "data-version": "33"